charts/mimir-distributed/Chart.yaml

```
@@ -28,3 +28,8 @@ dependencies:
     version: 8.0.10
     repository: https://helm.min.io/
     condition: minio.enabled
+  - name: rollout-operator
+    alias: rollout_operator
+    version: 0.1.0
+    repository: https://grafana.github.io/helm-charts
+    condition: rollout_operator.enabled
```

```suggestion
    condition: global.rollout_operator.enabled
```

Also will need OR per service as in store_gateway.rollout_operator.enabled or something like that